ci/build-docs.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-tracked_markdown=$(git ls-tree -r HEAD --name-only docs | egrep '\.(markdown|md)$')
+git ls-tree -r HEAD --name-only docs | egrep '\.(markdown|md)$' > tracked_markdown
```

Comment:
Let's use something like:

```bash
tracked_markdown=$(mktemp)
trap "EXIT" "rm ${tracked_markdown}"
```

and then use `$tracked_markdown` for the filename, so that this list is out of the way of the repo